build.gradle

```diff
@@ -105,6 +105,7 @@ repositories {
 dependencies {
     implementation "org.opensearch:opensearch:${opensearch_version}"
     implementation 'org.junit.jupiter:junit-jupiter:5.10.0'
+    implementation "org.apache.logging.log4j:log4j-slf4j-impl:2.19.0"
```

Comment:
This does nothing without slf4j-api.  We already inherit log4j2 from OpenSearch so we just need to configure the logger(s).  We should use the OpenSearch config like other plugins.